docs/filesystem.mdx

```diff
@@ -36,7 +36,7 @@ Penggunaan suatu file dapat dilakukan oleh program juga. Sehingga tidak hanya pe
 
 ## Folder
 
-Folder adahal suatu "kotak" virtual yang digunakan untuk menyimpan file-file yuang berkaitan. Tidak hanya itu, suatu folder bisa menyimpan folder lain, dimana folder tersebut dinamakan _sub-folder_. Pada umumnya, kamu ingin membuat suatu folder memiliki isi yang berkaitan dengan satu sama lain.
+Folder adahal suatu "kotak" virtual yang digunakan untuk menyimpan file-file yang berkaitan. Tidak hanya itu, suatu folder bisa menyimpan folder lain, dimana folder tersebut dinamakan _sub-folder_. Pada umumnya, kamu ingin membuat suatu folder memiliki isi yang berkaitan dengan satu sama lain.
```

Comment:
Hello, thanks for the PR ya! Any help on this repo is much appreciated especially since we haven't really setup a proper contribution guide yet 😅

Maybe nitip fix typo ini juga ya sekalian di PRnya, after that I can merge it 👍 
```suggestion
Folder adalah suatu "kotak" virtual yang digunakan untuk menyimpan file-file yang berkaitan. Tidak hanya itu, suatu folder bisa menyimpan folder lain, dimana folder tersebut dinamakan _sub-folder_. Pada umumnya, kamu ingin membuat suatu folder memiliki isi yang berkaitan dengan satu sama lain.
```

Reply:
On second thought it seems bisa gua commit directly suggestionnya, langsung gua merge ya 👍